client/settings/fraud-protection/index.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { __ } from '@wordpress/i18n';
+import { Card } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import CardBody from '../card-body';
+import { ProtectionLevels } from './components';
+import './style.scss';
+
+const FraudProtection = () => {
+	return (
+		<Card className="fraud-protection">
+			<CardBody>
+				<h4>{ __( 'Payment risk level', 'woocommerce-payments' ) }</h4>
```

Comment:
@bradyv Can we get your input here for the Fraud Protection card title? 

Should it be: Payment risk level

Or should it be: Set your payment risk level

See comment above for additional info.

Reply:
Sorry for the discrepancy, that's my fault for missing a few instances when updating the doc copy. It should be `Set your payment risk level` and I've updated the Figma file to match

Reply:
@allie500 Sorry for the discrepancy, that's my fault for missing a few instances when updating the doc copy. It should be `Set your payment risk level` and I've updated the Figma file to match

Reply:
@bradyv No worries! Thanks for clarifying. I'll get it updated.

Reply:
I've updated the section's card title to `Set your payment risk level` in [d53bc92](https://github.com/Automattic/woocommerce-payments/pull/5450/commits/d53bc922049dc4a4c856a8d0fbb55386d75cacdc).

However after I pushed that commit and reviewed Figma, I noticed an inconsistency in the setting's description to the left of the card, under the header `Fraud Protection`. @bradyv could you review the two items below and let me know which one is the correct one to use?

On this Figma layout (WIM5w2Y10SCNBFdMLSVftj-fi-1300%3A197505) it reads:
> Manage security and fraud prevention options to avoid chargebacks, unauthorized access to the store, and better store management experience.

On this Figma layout (WIM5w2Y10SCNBFdMLSVftj-fi-1300%3A197765) it reads: 
> Help avoid chargebacks by setting your security and fraud protection risk level.

Reply:
hey @allie500 

This is the correct copy for the setting's description under `Fraud protection`:

> Help avoid chargebacks by setting your security and fraud protection risk level.

Reply:
Just a quick note that the headers should all be sentence case (only the first word capitalized).